packages/ui-components/src/Icon/Icon.module.scss

```diff
@@ -1,12 +1,15 @@
 @import "../styles/tokens.scss";
-
 .icon {
-  display: inline-flex;
+  fill: $crl-base-text--dark;
   margin: auto;
 
   &:disabled {
     pointer-events: none;
   }
+
+  &__container {
```

Comment:
The Icon component should not have a container. A container can be provided by whoever is implementing it. The component should only have considerations for the SVG it is rendering. We want this component to consider itself like an image.

Reply:
Alright, makes sense. Updated.